src/index.js

```diff
@@ -83,8 +83,8 @@ module.exports = function createService(url, socketOptions) {
       async _setup() {
         try {
           this.queues = {};
-          const { queues, exchanges, channel = {} } = this.schema;
-          await this.channel.prefetch(channel.prefetch || 1);
+          const { queues, exchanges, channelOpts = {} } = this.schema;
+          await this.channel.prefetch(channelOpts.prefetch || 1);
```

Comment:
is it reasonable renaming? won't it break client code?

Reply:
The problem is that it was not working at all ;) since this.channel were redefined with amqp channel object

Reply:
OMG, ok. thanks.